data/json/items/comestibles/carnivore.json

```diff
@@ -25,7 +25,8 @@
     "volume": 1,
     "price": 500,
     "spoils_in": 12,
-    "calories": 200
+    "calories": 200,
+    "proportional": { "vitamins": [ [ "calcium", 2.0 ], [ "iron", 0.5 ] ] }
```

Comment:
Are you sure ```proportional``` works with arrays?
I've tested this but got no changes (at least according to RDA info in inventory). Could you double check this?